source/Engine/Bytecode/StandardLibrary.cpp

```diff
@@ -14390,39 +14337,14 @@ VMValue SocketClient_WriteString(int argCount, VMValue* args, Uint32 threadID) {
 VMValue Sound_Play(int argCount, VMValue* args, Uint32 threadID) {
 	CHECK_AT_LEAST_ARGCOUNT(1);
 	ISound* audio = GET_ARG(0, GetSound);
-	float panning = GET_ARG_OPT(1, GetDecimal, 0.0f);
-	float speed = GET_ARG_OPT(2, GetDecimal, 1.0f);
-	float volume = GET_ARG_OPT(3, GetDecimal, 1.0f);
-	int channel = -1;
-	if (audio) {
-		AudioManager::AudioStop(audio);
-		channel = AudioManager::PlaySound(audio, false, 0, panning, speed, volume, nullptr);
-	}
-	return INTEGER_VAL(channel);
-}
-/***
- * Sound.Loop
- * \desc Plays a sound, looping back when it ends.
- * \param sound (Integer): The sound index to play.
- * \paramOpt loopPoint (Integer): Loop point in samples.
- * \paramOpt panning (Decimal): Control the panning of the audio. -1.0 makes it sound in left ear only, 1.0 makes it sound in right ear, and closer to 0.0 centers it. (0.0 is the default.)
- * \paramOpt speed (Decimal): Control the speed of the audio. > 1.0 makes it faster, < 1.0 is slower, 1.0 is normal speed. (1.0 is the default.)
- * \paramOpt volume (Decimal): Controls the volume of the audio. 0.0 is muted, 1.0 is normal volume. (1.0 is the default.)
- * \return Returns the channel index where the sound began to play, or <code>-1</code> if no channel was available.
- * \ns Sound
- */
-VMValue Sound_Loop(int argCount, VMValue* args, Uint32 threadID) {
-	CHECK_AT_LEAST_ARGCOUNT(1);
-	ISound* audio = GET_ARG(0, GetSound);
-	int loopPoint = GET_ARG_OPT(1, GetInteger, 0);
+	int loopPoint = GET_ARG_OPT(1, GetInteger, -1);
```

Comment:
I think `Sound.Play` should also accept `MUSIC_LOOP_DEFAULT` and `MUSIC_LOOP_NONE`... Though they should be renamed in that case. (Suggestion: `AUDIO_LOOP_DEFAULT` and `AUDIO_LOOP_NONE`)